### DISCUSSION TITLE
Create and validate a custom theme for primeng 16

### DISCUSSION THREAD

**Discussion by fperie** [primefaces/community#176] [PrimeNG] [answered]
Hello,

my company had a licence to download the primeng designer software to develop a custom theme.
I would like to develop a new version of my custom theme on primeng v16.

On this https://www.primefaces.org/store/designer.xhtml web page, we can't download the designer software and especially, 
he is mentioned : "New Designer Sales are not available at the moment."

How to do this today ?
[Sakai-ng](https://github.com/primefaces/sakai-ng) is it the new demo application ?

Now, I don't understand the different part on web site :

- **theme** : allow to custom the visual rendering of all components ?
 Why the web site https://designer.primeng.org/ is still available ?

 - **blocks** : is it a composition of primeng components ?
 
 - **templates** : What is it exactly ? I'am sorry but I'm confused on the semantic and the target on this part. Can you help me ? Sakai-ng is a free template ?

I'm very sorry but here, I'm lost.
Am I the only one asking these kinds of questions ?

I must create a new custom primeng theme for my enterprise and validate it.
How can I do ?

Thank's a lot for your answer,
Fabien

**Comment by The-Real-1** [comment]
Hi ,

I agree with this questioning @fperie .
I wrote some similar questions here : https://github.com/orgs/primefaces/discussions/109#discussioncomment-7196427

Hope we'll get some explanations about the future of the Designer feature.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

- Sakai-ng is a free template for a long time.
- Designer webpage is still live and usable. However, the version is up to v16.
- However, we've made the designer open source, you can clone the repo from https://github.com/primefaces/primeng-sass-theme
- Since we've made the designer open source, the current designer webpage will not receive an update.
- Designer sale is not available now because it's become an open-source project. Right now we're making an extended version of the designer you can see in the roadmap https://primeng.org/roadmap
- PrimeBlocks is a paid project that includes 400+ ready-to-use copy-paste UI elements, it's not a composition of PrimeNG Components. You can see the free examples here https://blocks.primeng.org/#/free
- For templates, only sakai-ng is free and others require one-time payment you can check all of them here https://www.primefaces.org/store/templates.xhtml, if you have one please download the latest version of yours from the store.
- If you have a template you can simply follow the guide in here to use open source designer. It's an update guide but you can update your styles as described here https://github.com/orgs/primefaces/discussions/130#discussioncomment-7130755

**Comment by autonomobil** [comment]
> Designer webpage is still live and usable. However, the version is up to v16.

Not any more. Any way to bring that back, atleast for v16?
